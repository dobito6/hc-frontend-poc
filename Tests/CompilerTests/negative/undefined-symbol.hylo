class A {}
fun f(x: A.B) { true }
