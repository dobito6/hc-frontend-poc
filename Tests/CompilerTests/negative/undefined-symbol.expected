negative/undefined-symbol.hylo:2.12-13: error: type 'A' has no member 'B'
fun f(x: A.B) { () }
           ^
